gfx/video_driver.c

```diff
@@ -2082,8 +2087,13 @@ void video_driver_save_settings(config_file_t *conf)
    if (!conf)
       return;
 
+#ifdef _XBOX
    config_set_bool(conf, "gamma_correction",
          global->console.screen.gamma_correction);
+#else
+   config_set_int(conf, "gamma_correction",
+         global->console.screen.gamma_correction);
+#endif
```

Comment:
Rather than having this check inline, would it be better design to have it in configur_set_int() instead?

Is it just that XBOX uses a boolean gamma correction, while every other platofrm uses ints?